code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -1051,6 +1052,26 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 			overlays_standing[FACEMASK_LAYER] = standing
 	apply_overlay(FACEMASK_LAYER)
 
+/mob/living/carbon/human/update_inv_neck()
```

Comment:
Please add an `update_hud_neck()`, similar to what the other update_inv_* procs have, and include it in this proc. This is necessary for observing to work properly.

Reply:
Should be fixed now